<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TaskDetailWindow</class>
 <widget class="QDialog" name="TaskDetailWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>950</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>测试任务详情</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>15</number>
   </property>
   <property name="leftMargin">
    <number>30</number>
   </property>
   <property name="topMargin">
    <number>25</number>
   </property>
   <property name="rightMargin">
    <number>30</number>
   </property>
   <property name="bottomMargin">
    <number>25</number>
   </property>
   <item>
    <widget class="QLabel" name="labelTitle">
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>任务名称</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBasic">
     <property name="title">
      <string>基本信息</string>
     </property>
     <layout class="QGridLayout" name="gridLayoutBasic">
      <property name="spacing">
       <number>20</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="labelTaskIdKey">
        <property name="text">
         <string>任务ID:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="labelTaskIdValue">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="labelStatusKey">
        <property name="text">
         <string>任务状态:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="labelStatusValue">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelUnitKey">
        <property name="text">
         <string>参试单位:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="labelUnitValue">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="labelStationKey">
        <property name="text">
         <string>控制站:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLabel" name="labelStationValue">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labelProtocolKey">
        <property name="text">
         <string>测试协议:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="labelProtocolValue">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="labelStartTimeKey">
        <property name="text">
         <string>开始时间:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QLabel" name="labelStartTimeValue">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupTestItems">
     <property name="title">
      <string>测试项目</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayoutItems">
      <item>
       <widget class="QTableWidget" name="tableTestItems">
        <property name="columnCount">
         <number>5</number>
        </property>
        <column>
         <property name="text">
          <string>项目ID</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>项目名称</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>测试标准</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>测试结果</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>测试时间</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupReport">
     <property name="title">
      <string>测试报告</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayoutReport">
      <item>
       <layout class="QGridLayout" name="gridLayoutReport">
        <property name="spacing">
         <number>20</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="labelReportIdKey">
          <property name="text">
           <string>报告ID:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="labelReportIdValue">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="labelTestResultKey">
          <property name="text">
           <string>测试结果:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QLabel" name="labelTestResultValue">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="labelPassRateKey">
          <property name="text">
           <string>通过率:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="labelPassRateValue">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLabel" name="labelGenerateUserKey">
          <property name="text">
           <string>测试人员:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QLabel" name="labelGenerateUserValue">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="labelSummaryKey">
        <property name="text">
         <string>测试总结:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="textSummaryContent">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutButtons">
     <item>
      <widget class="QPushButton" name="btnExport">
       <property name="text">
        <string>导出报告</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnClose">
       <property name="mouseTracking">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>关闭</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnClose</sender>
   <signal>clicked()</signal>
   <receiver>TaskDetailWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
